pyproject.toml

```diff
@@ -45,6 +46,7 @@ requires = ["poetry_core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.mypy]
+mypy_path = ["src"]
```

Comment:
We want to lint tests too.

Reply:
I think it will, mypy check from the root directory. Since the tests are in the root it should work. Anyway, if it's required, mypy will error out since we are on very strict settings.